_components/in-page-navigation/guidance/accessibility.md

```diff
@@ -1,5 +1,5 @@
-- **Allow keyboard navigation.** Users should be able to navigate between nav items by using the `Tab` key. They should also be able to activate a link when pressing `Enter` on their keyboard. Users should be able to activate hover and focus states with both a mouse and a keyboard.
+- **Allow keyboard navigation.** Users should be able to navigate between items by using the `Tab` key. They should also be able to activate a link when pressing `Enter` on their keyboard. Users should be able to activate hover and focus states with both a mouse and a keyboard.
 
-- **Enable tab order so  keyboard users access the in-page navigation before the main content section.** When a user tabs through a page that contains the in-page navigation component, they should find the in-page navigation before the `main` content. Since the in-page nav will most commonly exist to the right of the content (and follows the `main` element in the markup) this may seem like a tab-order error, but tabbing through the entire page before getting to in-page navigation links is not logical, creates confusion, and diminishes the user experience.
+- **Keyboard users should access the in-page navigation before the main content.** When a user tabs through a page that contains the in-page navigation component, they should find the in-page navigation _before_ the `main` content. Since the in-page navigation appears after the main content in the reading order, this may seem like a tab-order error. However, tabbing through the entire page before getting to in-page navigation links is not logical, creates confusion, and diminishes the user experience.
```

Comment:
Made some additional updates here because the the recommended markup shows the in-page nav element is added  _before_ the `main` markup, not after. 

Because of this, there is no need to ask users to force tab order with keyboard navigation by default, so I removed reference to this. However, if we wanted to be explicit, we can restore that copy.